modules/upload_bundles_test

```diff
@@ -0,0 +1 @@
+../tests/modules/upload_bundles_test
```

Comment:
This is a symlink that allows upload_bundles_test to be turned on during testing. Modules are not discoverable except in certain places...go figure.